package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "babel-plugin-emotion": "^9.2.10",
     "emotion": "^9.2.10",
+    "polished": "^2.2.0",
```

Comment:
Also for our spacings in the theme.js. I probably wouldn't use it for border radius or border width, but I think spacings are fair game for that.

Reply:
Font size i'd look to see what we currently do, unclear if those make sense to be in rem or not.

Reply:
Double checking and trying to let my brain play some catch up this morning, font-sizes are already in rem, maybe that's enough? I'm completely open for debate on wether paddings should be rem or not.

Reply:
That's correct, font-sizings are already in rem. We're using the rem values directly though, so I wasn't sure if you were suggesting we update the usage so that we use the `rem` helper to pass pixel values instead.

I think you're right about using `rem` for spacing as well.

Reply:
I'm going to handle that in a separate PR.

Reply:
PR to use rem helper for spacings - https://github.com/CatchRelease/arbor/pull/9